airbyte-integrations/connectors/source-pipedrive/source_pipedrive/manifest.yaml

```diff
@@ -411,3 +411,15 @@ spec:
         examples:
           - "2017-01-25 00:00:00Z"
         type: string
+      num_workers:
+        title: Number of concurrent workers
+        description: Number of concurrent workers to be used in the sync
+        type: integer
+        minimum: 1
+        maximum: 40
+
+# rate limiting https://pipedrive.readme.io/docs/core-api-concepts-rate-limiting
```

Comment:
can you add a bit more detail how we arrived at a default of 10 and a max of 40?

Based on their docs it sounds like a daily limit with a minimum of 30,000 tokens and a multiplier based on subscription tier. It would be great to detail this because the daily rate limit reset can lead to a lot of 429 errors especially if syncs happen multiple times a day

Reply:
updated